### DISCUSSION TITLE
Max Shift and Base Shift in Forge

### DISCUSSION THREAD

**Discussion by deniznasif** [lllyasviel/stable-diffusion-webui-forge#1851] [Q&A] [answered]
Hello, is there a way to access Max and Base Shift for Flux in Forge (as sliders in the UI) like you can in ComfyUI??
<img width="606" alt="Screenshot 2024-09-17 at 18 26 34" src="https://github.com/user-attachments/assets/227893a9-4fa9-47cc-895f-2799dcc000fd">

**Comment by Vagrin** [comment]
What do they do exactly? It's not clear to me.

**Reply by deniznasif** [reply]
Adjusting the settings combined with a very low CFG scale produces better photorealistic images in Flux

**Reply by Mol31** [reply, marked as answer]
You can get this setting plus CLIP separate prompts in this extension https://github.com/DenOfEquity/forgeFlux_dualPrompt

**Reply by deniznasif** [reply]
Nice. I'll check it out tonight. Thanks!

**Reply by Vagrin** [reply]
So, what's a good setting for the best photoreal images in Flux in your opinion? Save me a bunch of slow annoying testing on my slow rig, please.
